lib/libc/gen/errlst.c

```diff
@@ -36,7 +36,7 @@
  *   "changes": [
  *     "support"
  *   ],
- *   "change_comment": "EPROT, ENOMETHOD"
+ *   "change_comment": "EPROT"
```

Comment:
I'm assuming not updating the date is the right thing to do, so that any other commits since then don't get missed (though I suspect in reality Brooks just always looks at the diff?)

Reply:
Huh, I'll defer to @brooksdavis on that.  I suspect not updating it is correct as well, but I hadn't considered the date.

Reply:
Updating the date to match the commit means that my script to review changes will skip the file. At this point it doesn't matter much if we update or not as we're not reliably keeping things up to date. I've occasionally contemplated doing batch updates when merging, but never came up with a smart enough workflow to make it not take forever.